shepherd.js/scripts/postprocess.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
```

Comment:
Manually changing this stuff feels like a hack. I would like to know more about the problems people were experiencing and if it was just with node 16 or lower or what exactly was going on, since all this stuff worked for me in both CJS and ESM projects.

Reply:
In case it helps, my experience using Shepherd.js 12.0.1, TypeScript 5.4, and Node.js 18.18.2:

* CommonJS, `"module": "NodeNext", "moduleResolution": "NodeNext"` (aka `Node16`) - works
* ESM, `"module": "NodeNext", "moduleResolution": "NodeNext"` (aka `Node16`) - compile error, "Property 'Tour' does not exist on type 'typeof import("node_modules/shepherd.js/dist/shepherd")'.
* CommonJS, `"module": "CommonJS", "moduleResolution": "Node"` (aka `Node10`) - compile error, "There are types at 'node_modules/shepherd.js/dist/shepherd.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'."